src/lr3_comp.erl

```diff
@@ -50,7 +50,8 @@ compile_normal_app(AppInfo) ->
     AppDir       = rebar_app_info:dir(AppInfo),
     OtherSrcDirs = rebar_dir:src_dirs(Opts),
     SourceDirs   = lr3_comp_util:get_src_dirs(AppDir, ["src"] ++ OtherSrcDirs),
-    OutDir       = lr3_comp_util:relative_out_dir(AppInfo),
+    OutDir       = filename:join([rebar_app_info:out_dir(AppInfo),
+                                  "ebin"]),
```

Comment:
Actually I don't know why I did that. Looking at `lr3_com_util:relative_out_dir/1` is doing the exact same thing. 

I can change this line back and leave it as it was before.

As of rewriting. I think the compile plugin can be cut down in size depending on the scope of it. For example, is rebar3 lfe compile supposed to compile the entrie application or only lfe specific files? For example the app-file compilation is done by rebar3 anyway. And if it is working perhaps we should not change for the sake of change.